pkg/cmd/chat/message/message.go

```diff
@@ -0,0 +1,240 @@
+package message
+
+import (
+	"strings"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/GetStream/stream-cli/pkg/utils"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		sendCmd(),
+		getCmd(),
+		getMultipleCmd(),
+		partialUpdateCmd(),
+		deleteCmd()}
+}
+
+func sendCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "send-message --channel-type [channel-type] --channel-id [channel-id] --text [text] --user [user-id]",
+		Short: "Send a message to a channel",
+		Example: heredoc.Doc(`
+			# Sends a message to 'redteam' channel of 'messaging' channel type
+			$ stream-cli chat send-message --channel-type messaging --channel-id redteam --text "Hello World!" --user "user-1"	
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			chType, _ := cmd.Flags().GetString("channel-type")
+			chId, _ := cmd.Flags().GetString("channel-id")
+			user, _ := cmd.Flags().GetString("user")
+			text, _ := cmd.Flags().GetString("text")
+
+			msg, err := c.Channel(chType, chId).SendMessage(cmd.Context(), &stream.Message{Text: text}, user)
+			if err != nil {
+				return err
+			}
+
+			cmd.Printf("Message successfully sent. Message id: [%s]\n", msg.Message.ID)
+			return nil
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringP("channel-type", "t", "", "[required] Channel type such as 'messaging' or 'livestream'")
+	fl.StringP("channel-id", "i", "", "[required] Channel id")
+	fl.StringP("user", "u", "", "[required] User id")
+	fl.String("text", "", "[required] Text of the message")
```

Comment:
how about other flags?

Reply:
added attachment as well

Reply:
are we going to provide upload functionality to get an url ?

Reply:
sure, how about next sprint?

Reply:
yes, good to add to the list

Reply:
and we can accept upload here too, it would be easier to use

Reply:
✅

Reply:
looks good, just don't use `url` as a variable name since it's standard package name

Reply:
will fix it in a bit

Reply:
✅